LoopKit/Insulin/InsulinModelSettings.swift

```diff
@@ -117,6 +89,37 @@ extension InsulinModelSettings: RawRepresentable {
     }
 }
 
+extension InsulinModelSettings: Codable {
+    private enum CodingKeys: String, CodingKey {
+        case exponentialPreset, walsh
+    }
+    private struct Exponential: Codable {
+        let preset: ExponentialInsulinModelPreset
+    }
+    private struct Walsh: Codable {
+        let value: WalshInsulinModel
+    }
```

Comment:
Why only make these `Codable` when within `InsulinModelSettings`?

Reply:
Because I resolved the merge wrong.  Now it should be ok !